ETC.tsv

```diff
@@ -5158,7 +5158,7 @@ ETC_20150317_005157	Defeat Biteregina!
 ETC_20150317_005158	Small Hive
 ETC_20150317_005159	We are too late!
 ETC_20150317_005160	Wounded Villager
-ETC_20150317_005161	So you followed me here...
+ETC_20150317_005161	Were you looking for me?...
```

Comment:
Except that this was the dialogue for the person you were looking for when the Elder said one of his villager didn't come back after checking whether the portal was working. The former line implies you followed him to where he was; which I find awkward because you never did follow him, you happened to only find him standing there.